ast/Helpers.h

```diff
@@ -317,6 +317,25 @@ class MK {
         return Hash(loc, std::move(keys), std::move(values));
     }
 
+    static bool isT(ExpressionPtr &expr) {
```

Comment:
I wanted to use this helper method in multiple modules within the `resolver.cc` file and wasn't sure where I should put them... so I put them here! Happy to move them.

Reply:
I think these methods can just live in `resolver.cc` as toplevel functions in the anonymous namespace.  Please move them into that file instead.